lib/statsd.rb

```diff
@@ -165,28 +220,28 @@ def select_shard(stat)
     end
   end
 
-  def signed_payload(key, message)
-    sha256 = Statsd.setup_openssl
-    payload = timestamp + nonce + message
-    signature = OpenSSL::HMAC.digest(sha256, key, payload)
-    signature + payload
-  end
+  class Buffer
+    DEFAULT_BUFFER_CAP = 512
```

Comment:
@jnunemaker: Although y'all in dotcom usually lean towards long PR bodies, in Systems (given that we contribute a lot of code upstream) we go for small, well defined commits and detailed commit messages: https://github.com/github/statsd-ruby/commit/a06bb3ad2128ef03711a38639ae338cab03afc4b ;)